core/src/replay_stage.rs

```diff
@@ -885,7 +893,12 @@ impl ReplayStage {
         progress_map: &ProgressMap,
         retransmit_slots_sender: &RetransmitSlotsSender,
         skipped_slots_info: &mut SkippedSlotsInfo,
+        has_voted: bool,
```

Comment:
nit: rename `has_voted` here and in other use cases to `has_new_vote_been_rooted` to be more explicit about what it represents